audio-worklet/src/noisegate-script-processor.js

```diff
@@ -56,19 +56,14 @@ class NoiseGate {
     const bufferSize = options.bufferSize || 0;
     this.attack = options.attack || 0;
     this.release = options.release || 0;
-    
+
     // The time constant of the filter has been set experimentally to balance
     // roughly delay for high frequency suppression.
-    let timeConstant;
-    if (options.timeConstant != null)
-      timeConstant = options.timeConstant;
-    else
-      timeConstant = 0.0025;
+    let timeConstant = (typeof options.timeConstant === 'undefined') ?
+        0.0025 : options.timeConstant;
```

Comment:
What if I set `options.timeConstant = [1,2,3]`?

Don't think we need to fix this here in this PR, but if you're going to do checking here, maybe make it more robust?  Maybe check that options.timeConstant is a number (and finite?) and if it is, use it, otherwise use the default 0.0025?